### PR TITLE
Reword rc

### PR DESCRIPTION
**Pull request by knarrff**

### CODE REVIEW

**Comment by CaptainSifff** [comment]
I like what you have written, but I feel that we have lost the direct reference to the research cycle. I think this is important  to know that software can be a part in different stages, and also other people can utilize a specific software in different stages of their cycle, and hence have different requirements. As a side-note, I don't quite like that in the competency section we already strongly refer to people being fully-qualified researchers.

File: paper_teaching-learning-RSE.md [hunk 1]
```diff
@@ -195,8 +195,9 @@ A manifestation can also be the curiosity for new tools which is a great asset f
 Lifelong learning then becomes more bearable.
 
 #### Understanding the research cycle (RC)
```
Comment (jcohen02) [inline paper_teaching-learning-RSE.md:197]: Should this perhaps be "Understanding the research lifecycle (RC)"?

**Comment by jcohen02** [comment]
> I like what you have written, but I feel that we have lost the direct reference to the research cycle.

Likewise, I like the text here but I feel something is missing. If I understand correctly, this section is about highlighting why knowledge of the research lifecycle is an important competency for an RSE?

In this context, I feel we should perhaps be covering some of the following points:
 - Knowledge of the research process - come up with an idea or hypothesis, likely linked to some existing work, outputs, findings, etc., understand how that is developed, literature reviews, how funding may be obtained, the iterative process of finding a route to a solution (proving/disproving the hypothesis, developing some new knowledge or outcome, etc.).
 - Why understanding the above is important as an RSE.
 - Understanding how/where software fits into the lifecycle.
 - What an RSE gains from having this knowledge/understanding - how to work effectively with researchers, etc.
 - ...

**Comment by CaptainSifff** [comment]
An update, here?

File: paper_teaching-learning-RSE.md (hunk 1 above)
Comment (CaptainSifff) [reply paper_teaching-learning-RSE.md:197]: in my brain at least, software(the product) has a lifecycle, whereas research(performing the work) is more like a continuous cycle of reincarnation....

**Comment by CaptainSifff** [comment]
See #70

**Pull request merged**